internal/services/consumption/consumption_budget_resource_group_resource_test.go

```diff
@@ -404,12 +404,10 @@ resource "azurerm_consumption_budget_resource_group" "test" {
   }
 
   notification {
-    enabled   = true
-    threshold = 90.0
-    operator  = "EqualTo"
-    // We don't update the value of threshold_type because toggling between the two seems to be broken
-    // See the comment on threshold_type in the schema for more details
-    threshold_type = "Forecasted"
+    enabled        = true
+    threshold      = 90.0
+    operator       = "EqualTo"
+    threshold_type = "Actual"
```

Comment:
The default is `Actual` so should this be `Forecasted` so that we can validate this field can be updated?

Reply:
It should be `Actual`, so that the `_completeUpdate` test (line 92) updates the threshold from `Forecasted` (line 321) to `Actual` (line 410).

Reply:
Must've looked at the wrong config earlier, thanks